common/Helpers/RuntimeHelper.cs

```diff
@@ -17,4 +18,13 @@ public static bool IsMSIX
             return PInvoke.GetCurrentPackageFullName(ref length, null) != WIN32_ERROR.APPMODEL_ERROR_NO_PACKAGE;
         }
     }
+
+    public static bool RunningOnWindows11

```

Comment:
nit: Something feels wrong about this name, maybe "IsWindows11" ?

Reply:
TBH putting this here at all felt a little wonky, but I found this file and it seemed better than anything else I saw. I wanted to indicate that this method isn't really about the runtime, but the underlying platform. Maybe IsOnWindows11 is a little less clunky but still accurate?